deployer/README.md

```diff
@@ -492,6 +494,34 @@ To set the parameters:
       oc create -n logging -f -
     # this creates the daemonset and starts fluentd with the new params
 
+#### Secure Forward
+
+To be able to accommodate different possible configurations for aggregated logging
+stacks, yet still maintain a light container we are now providing the [secure forward](https://github.com/tagomoris/fluent-plugin-secure-forward)
+plugin to enable customers to forward Fluentd logs from OpenShift's Fluentd to
+an external aggregator which can interpret Fluentd's messages and can then further
+process logs and/or forward them to a log store.
+
+To set up `secure forward`, you need only update the `secure-forward.conf` section of
```

Comment:
"To set up `secure forward`, update the `secure-forward.conf` section of the config map provided...."  Not sure where its provided from....

In general, I would suggest writing less with using 'you' as its just the configuration and not necessarily this person's per se.